platform/mds/mds/src/main/java/org/motechproject/mds/repository/MotechDataRepository.java

```diff
@@ -70,6 +70,12 @@ public PersistenceManager getPersistenceManager() {
         return pm;
     }
 
+    public void evictAll() {
+        if (persistenceManagerFactory != null) {
+            persistenceManagerFactory.getDataStoreCache().evictAll();
```

Comment:
Not really, but if people will start using the method they won't probably notice the implication of clearing the entire cache (at least not right away)